content/en/docs/KRE/user/40_kais_runner_sdk/_index.md

```diff
@@ -9,11 +9,104 @@ description: >
 
 ## The Runner SDK
 
+KAI Server supports Python and Golang providing an SDK that allow users to receive,
+process and send messages between nodes.
+
+Thanks to the SDK, users can implement `handler` and `init` functions, which are explained below.
```

Comment:
how about instead of "which" we could use "these" or scrap down the sentence and use for example, "they consist of the following" or something like that. This phrase sounds a little off to me.